exporter/collector/metrics.go

```diff
@@ -937,9 +943,15 @@ func defaultGetMetricName(baseName string, _ pmetric.Metric) (string, error) {
 	return baseName, nil
 }
 
+// this function converts the OTEL spec metric to cloud monitoring
 func numberDataPointToValue(
 	point pmetric.NumberDataPoint,
+	metricKind metricpb.MetricDescriptor_MetricKind,
 ) (*monitoringpb.TypedValue, metricpb.MetricDescriptor_ValueType) {
+	supportedTypedValue, supportedValueType := convertMetricKindToSupportedGCMTypes(metricKind, point.Attributes().AsRaw())
+	if supportedValueType != metricpb.MetricDescriptor_VALUE_TYPE_UNSPECIFIED {
+		return supportedTypedValue, supportedValueType
+	}
```

Comment:
May be good to log a message here so the user can debug what is wrong so the user can figure out why the conversion didn't take

Reply:
Added a log line in the conversion function.